### PR TITLE
fix for blank pages with page numbering

### PR DESCRIPTION
**Pull request by radius314**

### CODE REVIEW

**Comment by radius314** [comment]
@pjaudiomv can you convert 22 to a meaningful constant?

**Comment by pjaudiomv** [comment]
is that what you meant?

**Comment by pjaudiomv** [comment]
also if someone adjusts the page margin top or bottom this number would have to adjust proportionately

**Comment by pjaudiomv** [comment]
this would accommodate page margin changes.

**Comment by radius314** [comment]
Yes.  What’s the unit of measurement?  Maybe you could include that in the constant name.  Also typically constants are named in all capital letters.

File: bmlt-meeting-list.php [hunk 1]
```diff
@@ -1208,10 +1208,24 @@ function bmlt_meeting_list($atts = null, $content = null) {
...
+      if ($this->options['margin_top'] == '5') {
+        $ph_footer_fix_top = 0;
+      }
+      if ($this->options['margin_bottom'] < '5') {
+        $ph_footer_fix_bot = 5 - $this->options['margin_bottom'];
+      }
+      if ($this->options['margin_top'] < '5') {
+        $ph_footer_fix_top = 5 - $this->options['top'];
+      }
+      $PH_FOOTER_MM = 22 + $ph_footer_fix_top + $ph_footer_fix_bot;
```
[the remark this answers is not in the document]
Comment (radius314) [reply bmlt-meeting-list.php]: @pjaudiomv have you seen this comment? ^^

[the remark this answers is not in the document]
Comment (pjaudiomv) [reply bmlt-meeting-list.php]: weird i replied but they all said pending

[the remark this answers is not in the document]
Comment (radius314) [reply bmlt-meeting-list.php]: You need to submit a review

[the remark this answers is not in the document]
Comment (radius314) [reply bmlt-meeting-list.php]: Can you set 22 as another constant as well?

File: bmlt-meeting-list.php [hunk 2]
```diff
@@ -1208,10 +1208,25 @@ function bmlt_meeting_list($atts = null, $content = null) {
...
+        $ph_footer_fix_top = 0;
+      }
+      if ($this->options['margin_bottom'] < '5') {
+        $ph_footer_fix_bot = 5 - $this->options['margin_bottom'];
+      }
+      if ($this->options['margin_top'] < '5') {
+        $ph_footer_fix_top = 5 - $this->options['top'];
+      }
+      $PH_FOOTER_MM_HEIGHT_ADJUST = 22;
+      $PH_FOOTER_MM = $ph_footer_mm_height_adjust + $ph_footer_fix_top + $ph_footer_fix_bot;
```
Comment (radius314) [inline bmlt-meeting-list.php]: need to reference the right variable name.  have you re-tested this after you made changes?

Comment (pjaudiomv) [reply bmlt-meeting-list.php]: I have not, im at airport with limited internet.

**Comment by radius314** [comment]
Do you know how to test bread locally?

**Comment by pjaudiomv** [comment]
Ah yes the docker....that would of made testing so much faster.

**Comment by radius314** [comment]
Yup, just run docker-compose up from the command line in the bread folder.
That should stand up a fresh WP instance and a MySQL Db.

**Pull request merged**